pkg/grpc/server.go

```diff
@@ -56,6 +56,7 @@ func init() {
 var (
 	log = logging.LoggerForModule()
 
+	// MaxMsgSizeSetting is the setting used for gRPC servers and clients to set maximum receive sizes.
 	maxMsgSizeSetting               = env.RegisterIntegerSetting("ROX_GRPC_MAX_MESSAGE_SIZE", defaultMaxMsgSize)
```

Comment:
I think the variable should be:
```suggestion
	MaxMsgSizeSetting               = env.RegisterIntegerSetting("ROX_GRPC_MAX_MESSAGE_SIZE", defaultMaxMsgSize)
```

Reply:
Done